src/registrar/templates/base.html

```diff
@@ -61,18 +61,13 @@
   <a class="usa-skipnav" href="#main-content">Skip to main content</a>
 
   {% if IS_DEMO_SITE %}
-    <section
-    class="usa-site-alert usa-site-alert--emergency usa-site-alert--no-icon"
-    aria-label="Site alert"
-    >
-      <div class="usa-alert">
-        <div class="usa-alert__body">
-          <p class="usa-alert__text">
-            <strong>TEST SITE</strong> - Do not use real personal information. Demo purposes only.
-          </p>
-        </div>
+      <div class="usa-alert usa-alert--warning usa-alert--no-icon">
+      <div class="usa-alert__body">
+        <p class="usa-alert__text">
+        <strong>BETA SITE:</strong> We’re building a new way to get a .gov. Take a look around, but don’t rely on this site yet. This site is for testing purposes only. Don’t enter real data into any form on this site. To learn about requesting a .gov domain, visit <a href="https://get.gov" class="usa-link">get.gov</a>
```

Comment:
This is just missing the link for the blog post. And I think the previous section tag was ok so we keep the alert aria-label: 

```    <section aria-label="Alert" >
      <div class="usa-alert usa-alert--warning usa-alert--no-icon">
        <div class="usa-alert__body">
          <p class="usa-alert__text">
            <strong>BETA SITE:</strong> We're building a <a href="https://get.gov/updates/2022/9/14/making-infrastructure-less-invisible/">new way to get a .gov</a>. Take a look around, but don't rely on this site yet. This site is for testing purposes only. Don’t enter real data into any form on this site. To learn about requesting a .gov domain, visit <a href="https://get.gov/">get.gov</a>.
          </p>
        </div>
      </div>
    </section>
```